recipes/libsolace/all/conanfile.py

```diff
@@ -0,0 +1,54 @@
+import os
+
+from conans import CMake, ConanFile, tools
+
+
+class LibsolaceConan(ConanFile):
+    name = "libsolace"
+    description = "High performance components for mission critical applications"
+    topics = ("HPC", "High reliability", "P10", "solace", "performance", "c++", "conan")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/abbyssoul/libsolace"
+    license = "Apache-2.0"
+    author = "Ivan Ryabov <abbyssoul@gmail.com>"
```

Comment:
We are not adding the author attribute to recipes as there could be multiple contributors to it. Authorship will be tracked by github in the git commits